tests/integration/test_pipeline_processor_pool_router_config.py

```
@@ -0,0 +1,770 @@
+        router = FileRouter()
+        with pytest.raises(KeyError):
+            router.remove_extension(".doesnotexist")
+
+    def test_get_extension_map_returns_copy(self) -> None:
+        from file_organizer.pipeline.router import FileRouter
+
+        router = FileRouter()
+        ext_map = router.get_extension_map()
+        ext_map[".pdf"] = None  # type: ignore[assignment]
+        assert (
+            router.route(Path("file.pdf")) != None
```

Ruff (0.15.9)

[error] Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)